mesheryctl/internal/cli/root/model/export.go

```diff
@@ -90,8 +90,7 @@ func export(modelName string, url string, output *outputDetail) error {
 
 	resp, err := utils.MakeRequest(req)
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+		return ErrExportModel(errors.New("an error occured on server during export process"), modelName)
```

Comment:
```suggestion
		return ErrExportModel(errors.New("an error occurred on server during export process - " + err.Error()), modelName)
```

Adding the exact error message as well from the server can be useful for debugging. 
(not very useful in this case when server just throws the endpoint, but can be helpful for other error logs from server)

Reply:
the error return by server is not useful that's why I removed it see below

```bash
Get "http://localhost:9081/api/meshmodels/export?name=apisix&output_format=yaml&file_type=oci&components=false&relationships=false&pagesize=all": EOF
```